### PR TITLE
Card color,label

### PR DESCRIPTION
**Pull request by teja538**
please review:
added background color feature

### CODE REVIEW

**Review by raieegulhane — COMMENTED** [review]
The background color feature is working perfectly.
Liked the UI.
Clean code.

File: src/backend/db/db.json [hunk 1]
```diff
@@ -3,51 +3,109 @@
...
-      "notes": "notes textlkjkfa"
+      "notes": "notes textlkjkfa",
+      "backgroundColor": "",
+      "label": ""
+    },
+    {
+      "id": "d3e91f1b-4121-42bb-81e4-2174c6ef9b01",
+      "title": "lksadf",
+      "notes": "",
+      "backgroundColor": "#59657a"
```
Comment (krgarima) [inline src/backend/db/db.json:14]: use CSS color variables

**Review by dusariudaykumar — APPROVED** [review]
The code is clean and your file structure is also good, everything is well structured. Good Work!

**Review by krgarima — COMMENTED** [review]
UI Suggestion: Increase the width and margin of the `Save` and `Update` button. 
Recommended margin: 10px auto;
padding: 8px 24px;
Modify accordingly. This is just to give an idea.

**Comment by teja538** [comment]
> UI Suggestion: Increase the width and margin of the `Save` and `Update` button. Recommended margin: 10px auto; padding: 8px 24px; Modify accordingly. This is just to give an idea.

yeah will change accordingly garima. Thank you

**Pull request merged**